components/common/src/ome/system/PreferenceContext.java

```diff
@@ -164,6 +164,8 @@ public boolean canRead(EventContext ec, String key) {
             return true;
         case admin:
             return ec.isCurrentUserAdmin();
+        case user:
+            /* TODO */
```

Comment:
I would think the appropriate logic is "is member of `user` group"

Reply:
That's a thought. I'll add it to the to-do list!

Reply:
Makes sense. Implementation looks to be a larger change than I'd initially hoped.